diff --git a/langchain_code_sync/backend/services/git_manager.py b/langchain_code_sync/backend/services/git_manager.py
index 25af08c962e68de21ba36aaeb6f44bd1c7325b11..b0b09a651d494e0c7a734ae41c70776c5fa3503f 100644
--- a/langchain_code_sync/backend/services/git_manager.py
+++ b/langchain_code_sync/backend/services/git_manager.py
@@ -3,7 +3,7 @@ import json
 import os
 from typing import Dict, List, Tuple, Optional
 import hashlib
-# 这是source的注释
+# 这是source的注释，啊啊啊
 
 
 class DiscoveryManager:
